<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminEvent</class>
 <widget class="QDialog" name="AdminEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="eventList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="searchLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="monthSearch">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
       </font>
      </property>
      <item>
       <property name="text">
        <string>January</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>February</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>March</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>April</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>June</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>July</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>August</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>September</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>October</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>November</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>December</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="weekSearch">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Week 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Week 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Week 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Week 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Week 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="yearSearch">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPushButton" name="searchButton">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
       </font>
      </property>
      <property name="text">
       <string>Search Date</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="filterLabel">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>280</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bebas Neue</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QComboBox" name="filterDropdown">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Recent to Oldest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oldest to Recent</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hours: High - Low</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hours: Low - High</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>45</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reportButton">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>125</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
